aws_ec2_assign_elastic_ip/command_line_options.py

```diff
@@ -39,6 +39,10 @@
     help=(
         'Turn on dry run mode. No address will be assigned,\n'
         'we will only print which we whould take'))
+PARSER.add_argument(
+    '--skip-verify',
+    help='Skip verification that the EIP was successfully associated',
+    action='store_false', default=True, dest='verify')
```

Comment:
`default=True` is redundant here, that would be the default since `action='store_false'`, right?

Reply:
Yep, updated the PR